docs/content/en/docs/pipeline-stages/builders/_index.md

```diff
@@ -41,24 +41,37 @@ Skaffold supports building artifacts in following execution contexts:
 
 ## Local Build
 Local build execution is the default execution context.
-Skaffold will use the build tools locally installed on your machine to execute the build.
+Skaffold will use the build tools (such as Docker, Bazel, Maven or Gradle) locally installed on your machine to execute the build.
 
 **Configuration**
 
 To configure the local execution explicitly, add build type `local` to the build section of `skaffold.yaml`
 
 ```yaml
 build:
-  local:
-    ...
+  local: {}
 ```
 
+The following options can optionally be configured:
+
 {{< schema root="LocalBuild" >}}
 
-If you are deploying to [local cluster]({{<relref "/docs/environment/local-cluster" >}}), you can additional set `push` to `false` to speed up builds.
+**Faster builds**
+
+If you are deploying to [local cluster]({{<relref "/docs/environment/local-cluster" >}}), 
+Skaffold will default `push` to `false` to speed up builds.
 
+Skaffold can also build multiple artifacts in parallel, by settings a value higher than `1` to `concurrency`.
+For local builds, this is however disabled by default since local builds could have side effects that are
```

Comment:
Do you think we should mention specific builders that are believed to be safe (docker, Bazel) or that aren't safe (and the circumstances)?